support-workers/src/main/scala/com/gu/support/workers/lambdas/SendAcquisitionEvent.scala

```diff
@@ -18,7 +18,7 @@ import ophan.thrift.{event => thrift}
 import scala.concurrent.ExecutionContext.Implicits.global
 
 class SendAcquisitionEvent(serviceProvider: ServiceProvider = ServiceProvider)
-    extends ServicesHandler[SendAcquisitionEventState, Unit] {
+    extends ServicesHandler[SendAcquisitionEventState, Unit](serviceProvider) {
```

Comment:
This class was not passing the `ServiceProvider` it receives in its constructor through to its super class. This made it impossible to mock services in tests.